pubsub/client.go

```diff
@@ -122,11 +124,12 @@ const (
 )
 
 // Creates a new Radix client and subscribes to it.
-func New(address string) *Client {
+func New(address, password string) *Client {
```

Comment:
Perhaps we could pass a parameter-struct that contains both (or only one) of these fields.  The runtime would transparently give the optional password field a the zero-value for its type, and you wouldn't have to specify an empty string every time you make redis clients.

I was thinking something like:
```go
package redis

type ConnectionParam struct {
        Address string
        Password string
}
```
```go
package main

import ...

func main() {
        redis := redis.New(&redis.ConnectionParam{
                Address: "localhost",
                // optional password
        })
}
```

Of course, this method would need to be changed to have the header `func (*redis.ConnectionParam) *Client`